modules/utils.js

```diff
@@ -136,6 +136,18 @@ const findScenario = (node, searchId) => {
   return children.scenario;
 };
 
+const bindToClass = (module, thisClass) => {
+  const that = thisClass;
+  Object.entries(module).forEach((method) => {
+    const [key, value] = method;
+    if (key === 'init') {
+      value.call(that);
```

Comment:
Looks like it is a side effect.
Can we just bind all methods in this utility and call necessaries in the class right after the bindings?